src/api.rs

```diff
@@ -421,6 +422,7 @@ pub struct Events {
 
 impl Events {
     #[inline]
+    #[allow(dead_code)]
     pub(crate) fn events_raw(&self) -> &[*const Event] {
         use std::slice;
         unsafe {
```

Comment:
I'd probably also mark this function as unsafe if we end up leaving it in the API (especially the public one), but technically we would need to bump the crate version for that as well, so it's fine to leave it as-is for now (maybe make an issue if so).

(same for the other public functions that use unsafe, if we haven't verified their safety -- although this is tending towards the "vst-sys" and "vst" crate split as mentioned in chat)

Reply:
Nevermind, let's not block this fix because of this change. Made #98 to track this instead.